### PR TITLE
Feature/#116 field definition support

### PR DESCRIPTION
**Pull request by leonel1524**
[#116](https://github.com/adempiere/adempiere-vue/issues/116) hi everyone, in this PR is added support to field definition, to display personals styles for records table in window.
image

### CODE REVIEW

File: src/components/ADempiere/ContextMenu/items.vue [hunk 1]
```diff
@@ -44,6 +42,7 @@ export default {
   },
   methods: {
     handleClick(item) {
+      console.log(item)
```
Comment (yamelsenih) [inline src/components/ADempiere/ContextMenu/items.vue]: Hey remove this log please

File: src/components/ADempiere/ContextMenu/contextMenuDesktop.vue [hunk 2]
```diff
@@ -35,6 +35,9 @@
               {{ $t('components.contextMenuDownload') }}
             </a>
           </el-menu-item>
+          <el-menu-item v-show="$route.name === 'Report Viewer'" index="9" @click="$router.push({ name: '8e5131dc-fb40-11e8-a479-7a0060f0aa01' })">
```
Comment (yamelsenih) [inline src/components/ADempiere/ContextMenu/contextMenuDesktop.vue]: What is it?

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/ContextMenu/items.vue (hunk 1 above)
Comment (leonel1524) [reply src/components/ADempiere/ContextMenu/items.vue]: resolved

File: src/components/ADempiere/ContextMenu/contextMenuDesktop.vue (hunk 2 above)
Comment (leonel1524) [reply src/components/ADempiere/ContextMenu/contextMenuDesktop.vue]: resolved

**Pull request merged**